rosdep/base.yaml

```
@@ -5891,6 +5891,11 @@ mosquittopp-dev:
   fedora: [mosquitto-devel]
   nixos: [mosquitto]
   ubuntu: [libmosquittopp-dev]
+mpfr:
+  arch: [mpfr]
+  debian: [libmpfr-dev]
+  fedora: [mpfr]
+  ubuntu: [libmpfr-dev] 
```

Thanks for the links!
```suggestion
  gentoo: [dev-libs/mpfr]
  ubuntu: [libmpfr-dev]
```
A trailing space after the `ubuntu` entry is causing the check error
```
$ python scripts/check_rosdep.py rosdep/base.yaml 
checking for trailing spaces...
  ERR: trailing space line 5898
```
There's also a Gentoo package for it.